src/internal/constants/uiConfig.ts

```diff
@@ -61,6 +61,14 @@ export const defaultFilters = {
     },
 };
 
+export const disabledExportResultConfig = {
+    exportResult: { supported: false },
```

Comment:
I would rename it to `export: { supported: true }` cause exportResult has different meaning from its purpose

Reply:
updated